### DISCUSSION TITLE
How to override theme on edited cell?

### DISCUSSION THREAD

**Discussion by idafoh** [glideapps/glide-data-grid#338] [Q&A] [answered]
I want to change text color on edited cells how can I reach something like that?

**Comment by jassmith** [comment, marked as answer]
`getCellContent` should return a `themeOverride` for the cell when it is edited.

**Reply by idafoh** [reply]
Yeah I understand and I am using it now but when I edit width `onCellEdited` it should return `void` I want here change theme. Basically what I want is I set some color when initialized `getCellContent` then whenever it edited with `onCellEdited` I want to change it's text color to red.

**Reply by jassmith** [reply]
After a cell is edited the data grid will call getCellContent to get the updated content for that cell. You should return the new theme in there. The data grid does not maintain an internal cache of your data.

**Reply by idafoh** [reply]
Okay I understand
